packages/authentication/src/providers/authenticate.ts

```diff
@@ -4,7 +4,8 @@
 // License text available at https://opensource.org/licenses/MIT
 
 import * as http from 'http';
-import {HttpErrors, inject, ParsedRequest} from '@loopback/core';
+import {HttpErrors, ParsedRequest} from '@loopback/rest';
```

Comment:
Should we have `@loopback/http` for HTTP related classes/interfaces? I'm thinking about what an extension developer has to do if he/she wants to add `jsonrpc` support for LB Next.

Reply:
+1, I would say we can further refactor that out into its own library, if desired.

Reply:
+1, I think it may be better to do that later, out of scope of this PR?